x/leverage/keeper/collateral.go

```diff
@@ -78,9 +78,67 @@ func (k Keeper) GetBorrowerCollateral(ctx sdk.Context, borrowerAddr sdk.AccAddre
 	return totalCollateral
 }
 
-// GetEligibleLiquidationTargets returns a list of borrower addresses eligible for liquidation
+// addressExistInList returns true if address exist in the list of addresses.
+func addressExistInList(list []sdk.AccAddress, addrToCheck sdk.AccAddress) bool {
+	for _, addr := range list {
+		if addrToCheck.Equals(addr) {
+			return true
+		}
+	}
+
+	return false
+}
+
+// GetEligibleLiquidationTargets returns a list of borrower addresses eligible for liquidation.
 func (k Keeper) GetEligibleLiquidationTargets(ctx sdk.Context) ([]sdk.AccAddress, error) {
-	// TODO: Complete this to support liquidation tool
 	// ref: https://github.com/umee-network/umee/issues/229
-	return nil, nil
+	store := ctx.KVStore(k.storeKey)
+	borrowPrefix := types.CreateLoanKeyNoAddress()
+
+	iter := sdk.KVStorePrefixIterator(store, borrowPrefix)
+	defer iter.Close()
+
+	addressEligibleToLiquidation := []sdk.AccAddress{}
```

Comment:
Can rename this to `liquidationTargets`

Reply:
Done at https://github.com/umee-network/umee/pull/385/commits/26dd66a11b4d5ba964b3be3e6c0d2394d304b708